src/components/create-password/create-password-controller.ts

```diff
@@ -11,6 +11,12 @@ import {
 } from "../../utils/validation";
 
 export function createPasswordGet(req: Request, res: Response): void {
+  if (
+    req.session.user.featureFlags &&
```

Comment:
Is this 'double-boolean' approach preferable to using optional chaining operator (`?.`) for keeping the second check safe against undefined parents?

Reply:
updated